webview-ui/src/manualTest/clusterPropertiesTests.tsx

```diff
@@ -131,6 +133,31 @@ export function getClusterPropertiesScenarios() {
             agentPoolProfile.provisioningState = "Canceled";
             webview.postGetPropertiesResponse(testClusterInfo);
         }
+
+        async function handleAbortClusterOperation(hasError: boolean) {
+            await new Promise(resolve => setTimeout(resolve, 1000));
+            if (hasError) {
+                webview.postErrorNotification("Sorry, I wasn't quite able to abort the operation.");
+                return;
+            }
+
+            testClusterInfo.provisioningState = "Succeeded";
```

Comment:
Can we check what actually happens to the provisioning and power states when we abort a cluster operation, and make this consistent?

I _think_ that nothing changes immediately after sending the abort request, and then a little while later the cluster-level provisioning state changes to "Canceled". I'm not sure what happens at the agent pool level, but hopefully something similar?

It might make sense to avoid changing the power state at all here?